packages/wrangler/src/deployment-bundle/find-additional-modules.ts

```diff
@@ -0,0 +1,118 @@
+import { readdir, readFile } from "node:fs/promises";
+import path from "node:path";
+import chalk from "chalk";
+import globToRegExp from "glob-to-regexp";
+import { logger } from "../logger";
+import { parseRules, RuleTypeToModuleType } from "./module-collection";
+import type { Rule } from "../config/environment";
+import type { Entry } from "./entry";
+import type { CfModule } from "./worker";
+
+async function getFiles(root: string, relativeTo: string): Promise<string[]> {
+	const files = [];
+	for (const file of await readdir(root, { withFileTypes: true })) {
+		if (file.isDirectory()) {
+			files.push(...(await getFiles(path.join(root, file.name), relativeTo)));
+		} else {
+			// Module names should always use `/`. This is also required to match globs correctly on Windows. Later code will
+			// `path.resolve()` with these names to read contents which will perform appropriate normalisation.
+			files.push(
+				path
+					.relative(relativeTo, path.join(root, file.name))
+					.replaceAll("\\", "/")
```

Comment:
This is replacing all `\` chars with `/`. You'd need this instead:
```suggestion
					.replaceAll("\\\\", "/")
```

Reply:
This is the same code that is production right now. I think it is doing the right thing...